server/src/main/java/org/elasticsearch/cluster/metadata/MetaData.java

```diff
@@ -1206,6 +1241,10 @@ public static void toXContent(MetaData metaData, XContentBuilder builder, ToXCon
             builder.field("version", metaData.version());
             builder.field("cluster_uuid", metaData.clusterUUID);
 
+            builder.startObject("coordination");
```

Comment:
I wonder if we should call this `cluster_coordination` (similar to the `cluster_uuid` field above it). It's a little longer, but more explicit.

Reply:
https://github.com/elastic/elasticsearch/pull/35625/commits/d99c52d27586ef04a4ce64d3ce4daf6ef471cf73